tests/databases/redis/test_cluster.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+import time
+
+import redis as redisdb
+
+from testsuite.databases.redis import service
+
+
+CLUSTER_INIT_TIMEOUT = 5.0
+
+
+@pytest.fixture(scope='session')
+def _redis_service_settings(pytestconfig):
+    return service.get_cluster_service_settings()
+
+
+def test_cluster_basic(redis_service, _redis_service_settings: service.ServiceSettings):
+    def cluster_nodes():
```

Comment:
issue cluster specific command here

Reply:
`CLUSTER NODES` is a cluster specific command